app/middlewares/validator.js

```diff
@@ -0,0 +1,18 @@
+const { checkSchema, validationResult } = require('express-validator');
+
+const validate = (req, res, next) => {
+  const validationErrors = validationResult(req);
+  if (validationErrors.isEmpty()) {
+    return next();
+  }
+  const extractedErrors = [];
+  validationErrors.array().map(err => extractedErrors.push({ [err.param]: err.msg }));
```

Comment:
:thinking: This works but `map` is used to map elements, not to accumulate them.

Reply:
Changed the behavior of `map` in latest commit!